src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Items/Items.tsx

```diff
@@ -0,0 +1,103 @@
+import { Radio, Checkbox } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+
+import {
+  ItemConfigurationSettings,
+  SelectionRowsItem,
+  SelectionRowsOption,
+} from 'modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.types';
+import { UploaderUiType, Uploader, Svg } from 'shared/components';
+import { InputController } from 'shared/components/FormComponents';
+import { StyledFlexTopCenter } from 'shared/styles';
+
+import {
+  StyledSelectionRowItem,
+  StyledItemContainer,
+  StyledRemoveItemButton,
+} from './Items.styles';
+import { StyledSelectionBox } from '../SelectionRows.styles';
+
+const commonUploaderProps = {
+  width: 5.6,
+  height: 5.6,
+  uiType: UploaderUiType.Secondary,
+};
+
+export const Items = ({ isSingle }: { isSingle?: boolean }) => {
+  const { t } = useTranslation('app');
+
+  const { watch, control, setValue } = useFormContext();
+
+  const items = watch('selectionRows.items');
+  const options = watch('selectionRows.options');
+  const settings = watch('settings');
+
+  const hasTooltips = settings?.includes(ItemConfigurationSettings.HasTooltips);
+  const hasScores = settings?.includes(ItemConfigurationSettings.HasScores);
+  const hasRemoveButton = items?.length > 1;
+
+  const handleRemoveItem = (index: number) =>
+    setValue(
+      'selectionRows.items',
+      items?.filter((item: SelectionRowsItem, key: number) => key !== index),
+    );
+
+  return items?.map((item: SelectionRowsItem, index: number) => {
+    const name = `selectionRows.items[${index}]`;
+
+    return (
+      <StyledSelectionRowItem key={`row-${item.id}`} hasTooltips={hasTooltips}>
+        <StyledSelectionBox>
+          <StyledFlexTopCenter sx={{ gap: '1.2rem' }}>
+            <Uploader
+              {...commonUploaderProps}
+              setValue={(val: string) => setValue(`${name}.image`, val)}
+              getValue={() => watch(`${name}.image`) || ''}
+            />
+            <InputController
+              control={control}
+              name={`${name}.label`}
+              placeholder={t('selectionRowsItemPlaceholder', { index: index + 1 })}
+              maxLength={11}
+            />
+          </StyledFlexTopCenter>
+          {hasTooltips && (
```

Comment:
pls add possibility to add tooltip to the element

Reply:
will be added with the next PR (Item Settings Review)